examples/tv-app/linux/include/cluster-helper.cpp

```diff
@@ -0,0 +1,39 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <app/util/af-types.h>
+
+namespace {
+template <size_t BufferLength>
```

Comment:
this should be a header file instead of a c++ file.

consider naming it `zcl-strings.h` or similar. "helper" is to vague and such names end up having everything dumped into them.

Reply:
Yup. Will do

Reply:
Updated.